.github/workflows/codeql.yml

```diff
@@ -2,9 +2,9 @@ name: "CodeQL"
 
 on:
   push:
-    branches: [ "release/**", "develop", "master" ]
+    branches: [ "release/**", "develop", "master", "main" ]
   pull_request:
-    branches: [ "release/**", "develop", "master" ]
+    branches: [ "release/**", "develop", "master", "main" ]
```

Comment:
Could we remove master and develop from this list?

Reply:
I was thinking that but figured that `develop` might come back one day so didn't want to mess with anything for now. I don't think there is any harm in leaving those in.